src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/packages.config

```diff
@@ -1,6 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.ContainerRegistry" version="1.2.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.ContainerRegistry" version="2.0.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.6.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Storage" version="6.5.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
@Jinhuafei same comment about removing the added references in this file since they are already declared in a separate common project

Reply:
Several other packages depend on this package. Can't remove it from the packages.config.

Reply:
@Jinhuafei these packages are already found in the [`packages.config`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ResourceManager.Common/packages.config) file for the `Commands.ResourceManager.Common` project that _all_ cmdlet projects reference, so when the project is built, it will pull the references from NuGet first from the common project.

Reply:
Each project has its own packages.config which specifies the compile time and runtime dependency. Commands.ContainerRegistry project does depend on Commands.ResourceManager.Common project, but that doesn't mean Commands.ContainerRegistry project will inherit all the package reference from Commands.ResourceManager.Common project. At compile time, Commands.ContainerRegistry project only references Commands.ResourceManager.Common.dll, not including all the libs referenced by Commands.ContainerRegistry project.

Reply:
@Jinhuafei the `packages.config` file should have no effect on the compile-time or runtime dependency of a project - that information is contained in the `.csproj` file for a project. The `packages.config` file declares what version of a package should be pulled from NuGet when building the project. All cmdlet `.csproj` files import the [`Common.Dependencies.targets`](https://github.com/Azure/azure-powershell/blob/preview/tools/Common.Dependencies.targets) file, which adds the project dependency on these packages, and then those packages are pulled from NuGet when the `Commands.ResourceManager.Common` project is built.

Reply:
Looks like the reference of Microsoft.Rest.ClientRuntime is added not through Nuget package installation. But still it's not convenient to maintain the packages.config. Whenever I update nupkg through Nuget Package Manager, it may add those nupkgs as dependency. And I need to remember to delete these nupkgs from the file.

Also, if you search these 3 nupkg in the repo, you will find they are in bunch of packages.config.

Reply:
@Jinhuafei yes, we are usually making this comment in a lot of PRs because those references get added whenever someone messes with package manager, but it's always easy to spot during code review for us, so if you don't remember to delete them, we'll make a comment on the PR.

Those packages are found in a bunch of `packages.config` files in the repository, but those are for test projects, which don't use `Common.Dependencies.targets` because they don't ship with Azure PowerShell. We only maintain the versions of common packages that ship with our cmdlets. In the future we are going to be looking at creating a similar `.targets` file that all test projects use, and they will follow the same pattern as the cmdlet projects.